front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/ChartWidgets/ActiveUsersWidget/ActiveUsersCard.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useRef } from 'react';
+
+// hooks
+import useActiveUsers from 'modules/commercial/analytics/admin/components/ActiveUsersCard/useActiveUsers';
```

Comment:
There's a rule that we do not import anything within modules into anything outside the module, so this hook will need to be refactored to move into components/admin/GraphCards/